Lib/http/cookies.py

```diff
@@ -408,6 +408,8 @@ def OutputString(self, attrs=None):
                 append("%s=%s" % (self._reserved[key], _getdate(value)))
             elif key == "max-age" and isinstance(value, int):
                 append("%s=%d" % (self._reserved[key], value))
+            elif key == "comment" and isinstance(value, str):
+                append("%s=%s" % (self._reserved[key], _quote(value)))
```

Comment:
This patch looks like it correctly solves this problem, however is there a reason not to quote all values? (line417)

Reply:
Like I said in the issue (https://bugs.python.org/issue991266#msg315499) I'm a bit reluctant on that idea. The situation with cookie specs is a bit painful to deal :) Also, I'm not sure we can do that in bugfix releases because of backwards compatibility reasons.

Perhaps we can be a little bit conservative and merge this for now and discuss (like taking a look at other programming languages) a broader change like you and Mark mentioned in the issue for 3.8? I wrote this patch almost two years ago and to be honest I don't remember RFC 6265 and other relevant specs (2109, 2068 etc.) anymore :)

Reply:
Ok, I'm fine to 👍  this since it is correct.